sensorlib/config/noise_model_config.yaml

```diff
@@ -5,7 +5,9 @@ stages:
     type_noise: False
     list_inclusion_noise: False
 std_deviations:
+    # meters
     position: [0.8, 0.8, 0.8]
+    # radians
```

Comment:
I suggest appending either `_meters` or `_m` to these fields. That way the unit annotations follow the data.